app/uk/gov/hmrc/gform/sharedmodel/formtemplate/FormTemplate.scala

```diff
@@ -26,6 +26,7 @@ import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.Destinations
 case class FormTemplate(
   _id: FormTemplateId,
   originalId: FormTemplateId,
+  legacyFormIds: Option[NonEmptyList[FormTemplateId]],
```

Comment:
Integration test are not passing at the moment, due to this change, please run `it:test` and fix compilation errors.